deploy/sre-prometheus/README.md

```diff
@@ -0,0 +1,25 @@
+# Managed Cluster PrometheusRules
+
+This section contains alerts specifically crafted for managed OpenShift clusters.
+
+Alerts and recording rules have been evaluated not the be suitable to be contibuted to the
```

Comment:
nit "not the be" -> "to not be"

Reply:
Good catch, will fix that. It was pretty late already yesterday 😉